praca/praca.tex

```diff
@@ -2,21 +2,175 @@
+$$
+a_{ij}=a_{ji}.
+$$
+
+\noindent \textbf{Definicja 2.0.1.5}\\
+\noindent Macierz trójkątna to macierz kwadratowa, której wszystkie współczynniki pod główną przekątną lub wszystkie współczynniki nad tą przekątną są równe zero.\\
+
+\noindent \textbf{Definicja 2.0.1.6}\\
+\noindent Macierzą transponowaną (przestawioną) macierzy A nazywamy macierz $A^{T},$ która powstaje z danej poprzez zamianę jej wierszy na kolumny i kolumn na wiersze. Operację tworzenia macierzy transponowanej nazywamy transpozycją (przestawianiem).\\
+Dla macierzy $A=(a_{ij}):$ 
+$$ 
+\transpose{A} = \transpose{(a_{ij})}=(a_{ji})
+$$.
+
+\noindent \textbf{Definicja 2.0.1.7}\\
+\noindent Iloczynem macierzy $A = [a_{ij} ]$ i $B = [b_{jk}]$, gdzie $A\in M_{m}^{n}(F), $ $B\in M_{p}^{m}(F)$ nazywamy macierz $C = [c_{ik}], $ $C \in M_{p}^{n}(F), $ daną wzorem $$ c_{ik} = \sum_{j=1}^{n} a_{ij}\cdot b_{jk}. $$\\
+Oznaczamy C = A · B.\\
+
+\noindent \textbf{Definicja 2.0.1.8}\
+\noindent Niech A będzie macierzą kwadratową ustalonego stopnia. Macierz A jest odwracalna, jeśli istnieje taka macierz B, że zachodzi
+
+ $$A\cdot B=B\cdot A=I, $$ gdzie I jest macierzą jednostkową. Macierz B nazywa się wówczas macierzą odwrotną do macierzy A i oznacza się przez  $A^{-1}.$\\
+
+\noindent \textbf{Definicja 2.0.1.9}\\
+\noindent Macierz A nazywamy macierzą nieosobliwą, jeśli istnieje macierz B, która jest do niej odwrotna.\\  
+
+\noindent \textbf{Definicja 2.0.1.10}\\
+\noindent Macierzą ortogonalną nazywamy macierz kwadratową $A\in M_{n}(R)$ o elementach będących liczbami rzeczywistymi spełniająca równość:
+$$A^{T}\cdot A=A\cdot A^{T}=I_{n},$$ gdzie $I_{n}$ oznacza macierz jednostkową wymiaru n, $A^{T}$ oznacza macierz transponowaną względem A.\\
+
+\noindent \textbf{Definicja 2.0.1.11}\\
+\noindent Dwie macierze kwadratowe A i B nazywamy macierzami podobnymi, jeśli istnieje taka macierz nieosobliwa P, że zachodzi związek: $$ B=P^{-1}\cdot A\cdot P. $$
+
+
+\subsection{Definicje dotyczące wektorów} 
+
+\noindent \textbf{Definicja 2.0.2.1}\\
+\noindent Wektorem nazywamy uporządkowaną parę punktów, z których jeden jest początkiem a drugi końcem tego\\ wektora. Każdy wektor posiada zwrot, kierunek i długość.\\
+
+\noindent \textbf{Definicja 2.0.2.2}\\
+\noindent Układ wektorów  $ (v_{1}, v_{2}, ..., v_{k}) $ w przestrzeni liniowej $ V $ nad ciałem $ K $ nazywamy liniowo niezależnym, jeśli\\ z $ a_{1}\cdot v_{1}+ a_{2}\cdot v_{2} + ... + a_{k}\cdot v_{k} = 0 $ wynika, że $a_{1}=a_{2}=...=a_{k}=0 $.\\
+
+\subsection{Definicje dotyczące wyznacznika macierzy, jej wartości własnej oraz wektorów własnych.}
+
+\noindent \textbf{Definicja 2.0.3.1}\\
+\noindent Niech będzie dana macierz kwadratowa A stopnia n. Wyznacznikiem nazywamy takie odwzorowanie, które danej macierzy A wymiaru $n \times n$ przyporządkowuje dokładnie jedną liczbę rzeczywistą detA. Jeśli macierz jest stopnia $n = 1, $ to jej wyznacznik $detA = a_{11}$. 
```

Comment:
Dla det najlepiej jest zdefiniować własny operator/komendę
\newcommand{\det}{\operatorname{det}}
Będzie widać różnicę na ekranie, bo nie pozwoli to jej pochylanie.
Plus wsadzić to $ \det A $ w math-bloki